src/styles/getPopoverWithMeasuredContentStyles.ts

```diff
@@ -1,16 +1,9 @@
 import roundToNearestMultipleOfFour from './roundToNearestMultipleOfFour';
 import variables from './variables';
 
-/**
- * Compute the amount that the Context menu's Anchor needs to be horizontally shifted
- * in order to keep it from displaying in the gutters.
- *
- * @param {Number} anchorLeftEdge - Menu's anchor Left edge.
- * @param {Number} menuWidth - The width of the menu itself.
- * @param {Number} windowWidth - The width of the Window.
- * @returns {Number}
- */
-function computeHorizontalShift(anchorLeftEdge, menuWidth, windowWidth) {
+// Compute the amount that the Context menu's Anchor needs to be horizontally shifted
+// in order to keep it from displaying in the gutters.
+const computeHorizontalShift = (anchorLeftEdge: number, menuWidth: number, windowWidth: number): number => {
```

Comment:
```suggestion
function computeHorizontalShift(anchorLeftEdge: number, menuWidth: number, windowWidth: number): number {
```

NAB: In this case as you are typing the function directly, you can leave `function` signature.

Reply:
@hayata-suenaga any preference on this? Personally I like the function signature better, but not sure what we landed on?

Reply:
let's keep the function signature as @fabioh8010 suggested 🙇

Reply:
updated